public/docs/ts/latest/guide/structural-directives.jade

```diff
@@ -229,72 +250,68 @@ code-example(language="javascript").
     statement2;
     statement3;
   } 
+
 :marked
-  Without those braces JavaScript could only execute the first statement
+  Without those braces, JavaScript would only execute the first statement
   when you intend to conditionally execute all of them as a single block.
   The `<ng-container>` satisfies a similar need in Angular templates.
 
-a#asterisk
-.l-main-section
+.l-main-section#asterisk
 :marked
-  ## The asterisk (\*) prefix
+  ## The asterisk (*) prefix
   
-  Surely you noticed the asterisk (\*) prefix to the directive name
+  Surely you noticed the asterisk (*) prefix to the directive name
   and wondered why it is necessary and what it does.
   
   Here is `*ngIf` displaying the hero's name if `hero` exists.
 
-+makeExample('structural-directives/ts/src/app/app.component.html', 'asterisk')(format=".")
++makeExcerpt('src/app/app.component.html', 'asterisk', '')
+
 :marked
   The asterisk is "syntactic sugar" for something a bit more complicated.
   Internally, Angular "de-sugars" it in two stages.
   First, it translates the `*ngIf="..."` into a template _attribute_, `template="ngIf ..."`,&nbsp; like this.
-+makeExample('structural-directives/ts/src/app/app.component.html', 'ngif-template-attr')(format=".")
+
++makeExcerpt('src/app/app.component.html', 'ngif-template-attr', '')
+
 :marked
   Then it translates the template _attribute_ into a template _element_, wrapped around the host element, like this.
-+makeExample('structural-directives/ts/src/app/app.component.html', 'ngif-template')(format=".")
 
-// We should discourage writing <template> syntax in favor of <ng-container>
-  block remember-the-brackets
-    .alert.is-critical
-      :marked
-        Write `[ngIf]="hero"`, not `ngIf="hero"`!
-        The latter incorrectly assigns the *string* `"hero"` to `ngIf`.
-        A non-empty string is _truthy_, so `ngIf` is always `true` and Angular 
-        always tries to show the content … even when there is no `hero`.
++makeExcerpt('src/app/app.component.html', 'ngif-template', '')
 
 :marked
-  * The `*ngIf` directive moved to the `<template>` tag where it became a property binding,`[ngIf]`.
-  * The rest of the `<div>`, including its class attribute, moved inside the `<template>` tag.
+  * The `*ngIf` directive moved to the `<template>` element where it became a property binding,`[ngIf]`.
```

Comment:
Note to self:  all `<template>` will become `<ng-template>` in Angular v.4.   I have to think about what this does to what we've been saying here.